server/eclipse-project/src/main/java/us/freeandfair/corla/csv/ColoradoBallotManifestParser.java

```diff
@@ -126,6 +137,20 @@ public ColoradoBallotManifestParser(final String the_string,
   }
   
   /**
+   * Checks to see if the set of parsed manifests needs flushing, and does so 
+   * if necessary.
+   */
+  private void checkForFlush() {
+    if (my_parsed_manifests.size() % BATCH_SIZE == 0) {
```

Comment:
Would it be better to check `> BATCH_SIZE`? As written, this will not flush for any multiple of `BATCH_SIZE`.